Patches/Apex - Rimworld Legends/ApexRimworldLegends_Weapons_MachineGuns.xml

```diff
@@ -68,7 +68,7 @@
               </li>
               <li Class="CombatExtended.ToolCE">
                 <label>barrel</label>
-                <capacities>
+                <capacities>63/3
```

Comment:
Doesn't this give you any errors?

Reply:
oh my god i didnt realize i edited that in while i was typing in chrome